config/database.yml

```diff
@@ -20,10 +20,14 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV.fetch("DB_HOST", "localhost") %>
```

Comment:
Not entirely sure of the details here but rails doesn't check localhost for the database. `ENV.fetch` means we will check `ENV['DB_HOST']` for a value. If that env var doesn't exist we will default to `localhost`.

This line means that rails will check `localhost:<db port number>`, which is where docker will be exposing our database.